REMACS_COMPATIBILITY.md

```diff
@@ -4,13 +4,6 @@ Remacs should generally behave identically to GNU Emacs. If you find a
 difference, please
 [file a bug](https://github.com/Wilfred/remacs/issues/new).
 
-## Differences
-
-   - `base64-encode-string` and `base64-decode-string` does not fail
```

Comment:
Are we now erroring on multibyte strings? Whilst we're calling `error` in e.g. `base64_encode_string`, that was true before.